examples/next-starter/src/app/configs/appConfig.ts

```diff
@@ -1,9 +1,21 @@
+import { AppConfig } from '@onr/core';
 import getConfig from 'next/config';
+
 const { publicRuntimeConfig } = getConfig() as {
   publicRuntimeConfig: Record<'processEnv' | string, unknown>;
 };
 
-export const appConfig = {
+export const appConfig: AppConfig = {
   apiBaseUrl: `${publicRuntimeConfig.processEnv.NEXT_PUBLIC_API_URL}/api`,
+
   apiKey: publicRuntimeConfig.processEnv.NEXT_PUBLIC_API_KEY || '',
+
+  // prettier-ignore
+  'full-page-routes': [
```

Comment:
I feel "full-page-routes"  is a little ambiguous, how about add comment to describe it? like "routes not show on dashboard navbar"